src/components/ApplicationForm/Questionnaire.js

```diff
@@ -0,0 +1,152 @@
+import React from 'react'
+import styled from 'styled-components'
+import { Dropdown, Select, TextInput } from '../../components/Input'
+import { H1, QuestionHeading } from '../../components/Typography'
+
+const StyledDropdown = styled(Dropdown)`
+  .react-select__control {
+    margin: 0 0 1em;
+  }
+`
+
+const QuestionContainer = styled.div`
+  margin-bottom: 6em;
+`
+
+// form part 3
+export default ({ formInputs, onChange }) => {
+  const options = [
```

Comment:
also one last thing, do you mind moving this out of `export default ({ formInputs, onChange }) ` and add an export like this? @vincentchiang1 might need this map for the review page
```suggestion
  export const options = [
```